docs/content/docs/operators/feature/minhashlsh.md

```diff
@@ -0,0 +1,276 @@
+---
+title: "MinHash LSH"
+weight: 1
+type: docs
+aliases:
+- /operators/feature/minhashlsh.html
+---
+
+<!--
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+
+## MinHash LSH
+
+MinHash LSH is a Locality Sensitive Hashing (LSH) scheme for Jaccard distance metric.
+The input features are sets of natural numbers represented as non-zero indices of vectors,
+either dense vectors or sparse vectors. Typically, sparse vectors are more efficient.
+
+### Input Columns
+
+| Param name | Type   | Default   | Description            |
+|:-----------|:-------|:----------|:-----------------------|
+| inputCol   | Vector | `"input"` | Features to be mapped. |
+
+### Output Columns
+
+| Param name | Type          | Default    | Description  |
+|:-----------|:--------------|:-----------|:-------------|
+| outputCol  | DenseVector[] | `"output"` | Hash values. |
+
+### Parameters
+
+| Key                     | Default                                                   | Type    | Required | Description                                                        |
+|-------------------------|-----------------------------------------------------------|---------|----------|--------------------------------------------------------------------|
+| inputCol                | `"input"`                                                 | String  | no       | Input column name.                                                 |
+| outputCol               | `"output"`                                                | String  | no       | Output column name.                                                |
+| seed                    | `"org.apache.flink.ml.feature.lsh.MinHashLSH".hashCode()` | Long    | no       | The random seed.                                                   |
```

Comment:
I think `null` is ok for the default value of `seed`, just like in `KMeans` doc.

Reply:
Alright, I'll following existing example.